### ISSUE TITLE
normflux for flux_radius 

### ISSUE THREAD

**Issue by junohsu** [kbarbary/sep#67]
Hi,

Does "Normalizing flux for each position" mean total flux of each source? I am not sure I understand it correctly.

thanks,
Yun-Hsin

**Comment by kbarbary** [comment]
Yes. Am example: if you pass `normflux=10.0` and `frac=0.5`, the returned radius will enclose a flux of 5.
